java/clients/src/test/java/sleeper/clients/teardown/ShutdownSystemProcessesIT.java

```diff
@@ -215,7 +296,8 @@ void shouldTerminateEMRClustersInBatchesOfTen() throws Exception {
                     .willReturn(aResponse().withStatus(200).withBody(
                             "{\"Clusters\": []}"))
                     .whenScenarioStateIs("TERMINATED"));
-
+            stubFor(listActiveApplicationsRequest()
+                    .willReturn(aResponseWithNumRunningApplications(0)));
             // When
```

Comment:
This is missing an empty line before the When.

Reply:
Added